foo_005_rw2_crop_area_production/foo_005_rw2_crop_area_production_processing.py

```diff
@@ -37,71 +38,85 @@
 Download data and save to your data directory
 
 Dataset files can be downloaded at the following link:
-https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/PRFF8V&version=4.0
+https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/SWPENT
```

Comment:
I think we should specify the version number in the url. Looks like [&version=3.0](https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/SWPENT&version=3.0) is the latest